assets/css/main.css

```diff
@@ -58,6 +58,27 @@ table.src tr td.src {
   user-select: none;
 }
 
+/* focus styles (component-specific focus styles can be included in other 
+   components) */
+
+a:focus { /* Mostly for links in text */
+    background-color: #ffdd00 !important;
+    color: #333 !important;
+    outline: 3px solid rgba(0, 0, 0, 0);
+    text-decoration: underline;
+}
+
+a.btn:focus, button:focus, input.btn:focus {
```

Comment:
How does this work when one clicks a button and the button opens a modal dialog for example. Does that show the yellow highligh too? Is that ok and is there anything we could do about that?

Reply:
The highlight applies to the thing that has keybord focus. So, you focus on the button, then activate it. The modal appears, but initially nothing in the modal has keyboard focus. When you press tab, you jump to the next thing that should be reachable.

Ideally, this would be inside the model. We'd remove the page underneath from the tab order while it's unreachable. So maybe there are fields or links in the modal, or a close button. Tab should cycle the user around the visible, usable elements until they close the modal.

At present, I've not touched this and it doesn't work that way. Focus is kept by the button after the button is clicked. Pressing tab takes you further along the page. I recorded a 90-second example for you: https://aalto.cloud.panopto.eu/Panopto/Pages/Viewer.aspx?id=887cee86-b69b-4754-a897-abd100ca4658

The whole modal shouldn't get focus. 

In short, this PR only concerns how we show what currently has focus, not how we manage where the focus should be at a given moment.

Reply:
Thanks. That video was a good example (basically, how focus breaks). You could create an issue about that, or you can add tha to the list of issues for now. For sure, thing to look into later!

I was more considering a case where a person clicks "submit" with a mouse and, if I remember right, the "focus" flag is left in the button until something else takes it. However, I might be wrong here. Nevertheless, based on the video, I think this seems to work as expected (i.e. the submit loses focus and doesn't stay yellow).

Furthermore, if this change happens to introduce unintended consequences, then we can address those later. Benefits are good.